src/components/Toaster/Toast.tsx

```diff
@@ -0,0 +1,103 @@
+import { palette } from '../variables';
+import {
+  Toast,
+  ToastType,
+  resolveValue,
+  toast as toastEffect,
+} from 'react-hot-toast';
+import styled from 'styled-components';
+import { IconButton } from '../Buttons';
+import Text from '../Text';
+import { Warning } from '@/components/Icons/Warning';
+import { Success } from '@/components/Icons/Success';
+
+type Props = {
+  toast: Toast;
+};
+
+export const AppToast = ({ toast }: Props) => {
+  const handleClose = (id: string) => {
+    toastEffect.dismiss(id);
+  };
+
+  const getToastStyle = (type: ToastType) => {
+    switch (type) {
+      case 'error': {
+        return {
+          mainColor: palette.red,
+          Icon: <Warning color={'blueDark'} sizeInPx={23} />,
+        };
+      }
+
+      case 'success': {
+        return {
+          mainColor: palette.blue,
+          Icon: <Success color={'blueDark'} sizeInPx={23} />,
+        };
+      }
+
+      default: {
+        return {
+          mainColor: palette.purple,
+          Icon: <Warning color={'orange'} sizeInPx={23} />,
```

Comment:
Duud, why no 24 to support scaling better? Probably cos design but so close to a multiple of 8

Reply:
hehe I did actually convert lot of the values to the closest multiple of 8 but not this one, will change it